src/index.jsx

```diff
@@ -2,24 +2,23 @@ import { AppContainer } from 'react-hot-loader';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+
 const rootEl = document.getElementById('root');
 ReactDOM.render(
   <AppContainer>
     <App />
   </AppContainer>,
-  rootEl
+  rootEl,
 );
 
 if (module.hot) {
   module.hot.accept('./App', () => {
-    // If you use Webpack 2 in ES modules mode, you can
-    // use <App /> here rather than require() a <NextApp />.
-    const NextApp = require('./App').default;
+    const NextApp = require('./App').default; // eslint-disable-line global-require
```

Comment:
I tried to rewrite, but I couldn't get it to work... I'll see if I can remove the `default`.